### PR TITLE
Notebook 4

### PR DESCRIPTION
**Pull request by MartinSchobben**

### CODE REVIEW

File: unit_02/04_l-band-sar.ipynb [hunk 1]
```diff
@@ -4,17 +4,415 @@
    "cell_type": "markdown",
```
Comment (maybedave) [inline unit_02/04_l-band-sar.ipynb:4]: <p>we are using ALOS-2, not ALOS-1. The satellite is still operational. Pease update throughout the notebook.</p><p>We can eventually give out couple of more information about the input data:</p><ul><li>L-band: We are using The 25 m PALSAR-2 ScanSAR, which is a normalized backscatter data of PALSAR-2. We are using level 2.2 data, which are ortho-rectified and radiometrically terrain-corrected. They are provided in DN (digital numbers) and HH and HV polarization.</li><li>C-band: We are using 10 meters RTC gamma naught, only VV polarized.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TUW-GEO/microwave-remote-sensing/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (maybedave) [inline unit_02/04_l-band-sar.ipynb:4]: <p>γoE mean that is referred to the ellipsoid, but it is not. The data from ALOS are&nbsp;radiometrically terrain-corrected.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TUW-GEO/microwave-remote-sensing/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (MartinSchobben) [reply unit_02/04_l-band-sar.ipynb:4]: will ammend

Comment (MartinSchobben) [reply unit_02/04_l-band-sar.ipynb:4]: But is PALSAR not something different?

**Review by maybedave — APPROVED** [review]

**Pull request merged**